src/main/java/uk/ac/ebi/eva/submission/service/SubmissionService.java

```diff
@@ -0,0 +1,31 @@
+package uk.ac.ebi.eva.submission.service;
+
+import org.springframework.stereotype.Service;
+import uk.ac.ebi.eva.submission.model.SubmissionStatus;
+import uk.ac.ebi.eva.submission.repository.SubmissionRepository;
+
+import java.util.UUID;
+
+@Service
+public class SubmissionService {
+
+    private SubmissionRepository submissionRepository;
+
+    public SubmissionService(SubmissionRepository submissionRepository) {
+        this.submissionRepository = submissionRepository;
+    }
+
+    public SubmissionStatus createFtpDirectory() {
+        String submissionId = UUID.randomUUID().toString();
+
```

Comment:
I'm assuming actually creating the directory isn't in scope for the initial prototype...

Reply:
We talked about this on Friday and I think it should be and we can just have a property to set the base directory.
I think we should also change the permission of the directory as well to 620.

Reply:
added